pkg/util/k8sutil/k8sutil.go

```diff
@@ -372,6 +372,7 @@ func newEtcdPod(m *etcdutil.Member, initialCluster []string, clusterName, state,
 			},
 		},
 	}
+	setAnnotations(pod, cs.Pod.Annotations)
```

Comment:
All pod policy setting should be done in applyPodPolicy(): 
https://github.com/coreos/etcd-operator/blob/83d7ae64f92992b05818919fb682830225201bf1/pkg/util/k8sutil/pod_util.go#L91:6

This would also avoid the nil pointer

Reply:
Yep, I realized this right after I saw the failed tests. Fixed it in my latest commit.